src/components/webview/DiffWebviewPanel.ts

```diff
@@ -77,7 +77,10 @@ export class DiffWebviewPanel extends IntuitaWebviewPanel {
 		}
 
 		const job = this.__jobManager.getJob(jobHash);
-		const jobAccepted = this.__jobManager.isJobAccepted(jobHash);
+		// @TODO
```

Comment:
what about this TODO?

Reply:
We still need remove some code that was depending on jobAccepted, i will do this in next PR